### PR TITLE
Add the hideUnchangedPlanComments parameter

### PR DESCRIPTION
**Pull request by antham**
## what

Add a parameter to be able to set the `ATLANTIS_HIDE_UNCHANGED_PLAN_COMMENTS` environment variable that allows to remove no-changes plan comments from the pull request

## why

We already have `ATLANTIS_HIDE_PREV_PLAN_COMMENTS` available so it makes sense to have both

## tests

Tested locally by generating the template

### CODE REVIEW

**Comment by jamengual** [comment]
please bump the Chart minor version.
Thanks for the contribution @antham

**Comment by antham** [comment]
done @jamengual

File: charts/atlantis/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: v0.24.4
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.14.1
+version: 4.15.0
```
Comment (jamengual) [inline charts/atlantis/Chart.yaml:6]: lets do `4.14.2`

Comment (antham) [reply charts/atlantis/Chart.yaml:6]: Ah so the patch version

Comment (antham) [reply charts/atlantis/Chart.yaml]: done

**Review by jamengual — APPROVED** [review]

**Pull request merged**